GruntFile.js

```diff
@@ -125,6 +128,18 @@ module.exports = function(grunt) {
                     ]
                 }
             }
+        },
+        exec: {
+            "meteor-init": {
+                // Make sure that Meteor is installed.
+                command: "type meteor > /dev/null 2>&1 || { curl https://install.meteor.com/ | sh; }"
```

Comment:
whoa...    you should be using https://github.com/MeteorPackaging/grunt-gulp-meteor  I think, not downloading manually like that.